test/e2e/util/hypershift_framework.go

```diff
@@ -108,6 +108,9 @@ func (h *hypershiftTest) after(hostedCluster *hyperv1.HostedCluster, opts *core.
 		NoticePreemptionOrFailedScheduling(t, context.Background(), h.client, hostedCluster)
 		EnsureAllRoutesUseHCPRouter(t, context.Background(), h.client, hostedCluster)
 		EnsureNetworkPolicies(t, context.Background(), h.client, hostedCluster)
+		if platform == hyperv1.AWSPlatform {
+			CheckAffinityOpinions(t, context.Background(), h.client, hostedCluster)
```

Comment:
can we rename the function to be consistent with the other tests. something like `EnsureHCPPodsAffinitiesAndTolerations()`

lgtm otherwise

Reply:
amended latest commit with the recommended change